tests/unit/checkmate/views/api/check_url_test.py

```diff
@@ -39,6 +39,15 @@ def test_it_returns_an_error_for_no_url(self, make_request):
         with pytest.raises(BadURLParameter):
             check_url(request)
 
+    def test_it_returns_an_error_if_blocklist_raises_MalformedURL(
+        self, make_request, blocklist
+    ):
+        request = make_request("/api/check?url=http://")
+        blocklist.check_url.side_effect = MalformedURL("URL is bad")
```

Comment:
Looks like the test above, but doesn't actually trigger as the blocklist object is mocked out. The URL here is therefore just informational / decorative.